src/utils/URLUtils.ts

```diff
@@ -15,6 +15,17 @@
  */
 import { StringifiableRecord, stringifyUrl } from 'query-string';
 
+interface CortexScorecardServicePageURLProperties {
+  scorecardId: string | number;
+  serviceId: string | number;
+  cortexURL?: string;
+}
+
+interface CortexScorecardPageURLProperties {
+  scorecardId: string | number;
+  cortexURL?: string
+}
+
```

Comment:
Should this be? 

```suggestion
interface CortexScorecardServicePageUrlProperties {
  scorecardId: string | number;
  serviceId: string | number;
  cortexUrl?: string;
}

interface CortexScorecardPageUrlProperties {
  scorecardId: string | number;
  cortexUrl?: string
}
```

Reply:
I was sticking with the existing conventions in this repository, which capitalizes URL. If we want to apply the same naming conventions we use in our internal repos we should do that separately.

Reply:
It's kinda a mix right now though no? Like the var name is frontendBaseUrl

Reply:
That's an environment variable; not a code variable.